terraform/modules/emqx-ee/versions.tf

```diff
@@ -24,5 +24,10 @@ terraform {
       source  = "hashicorp/local"
       version = "2.4.0"
     }
+
+    null = {
```

Comment:
I don't see the corresponding lock file updates

Reply:
null was already part of the lock file under terraform/emqx